R/tm_g_association.R

```diff
@@ -336,7 +336,7 @@ srv_tm_g_association <- function(id,
       teal::validate_has_data(ANL[, c(ref_name, vars_names)], 3, complete = TRUE, allow_inf = FALSE)
 
       # reference
-      ref_class <- class(ANL[[ref_name]])
+      ref_class <- class(ANL[[ref_name]])[1]
```

Comment:
```suggestion
      ref_class <- class(ANL[[ref_name]])[1L]
```

Reply:
What difference does it make?  Indexing a small vector has no difference when it comes to what we use. I'd prefer to use just the `1` as it's easier on the eyes.
We use `[1]` in over 100 places in the repo. And a couple of places where we use `[1L]`.